src/store/Modules/Projects.ts

```diff
@@ -587,10 +591,11 @@ export default class Projects extends VuexModule {
       this.context.commit("updateConnection", connectionData)
     } catch (err) {
       console.log(err)
+      this.context.commit("updateConnection", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.